admin-auth/src/main/kotlin/com/epam/drill/admin/auth/model/AuthConfigModels.kt

```diff
@@ -0,0 +1,41 @@
+package com.epam.drill.admin.auth.model
+
+import com.epam.drill.admin.auth.config.AuthConfig
+import com.epam.drill.admin.auth.config.OAuth2Config
+import com.epam.drill.admin.auth.config.SimpleAuthConfig
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class AuthConfigView(
+    val simpleAuth: SimpleAuthConfigView? = null,
+    val oauth2: OAuth2ConfigView? = null
+)
+
+@Serializable
+data class SimpleAuthConfigView(
+    val enabled: Boolean,
+    val signUpEnabled: Boolean
```

Comment:
Can we have simple auth but disable signUp?
One case I see: migration from simple auth to oauth2 and external users, but we want to keep "old" users intact